scripts/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,39 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs';
+
+const getNodeReleasesData = async () => {
+  const nodevuOutput = await nodevu();
+
+  // Filter out those without documented support
```

Comment:
What are the unsupported versions? As on https://nodejs.org/en/download/releases we still show the version 0.12 for example.

Reply:
The ones not in [schedule.json](https://raw.githubusercontent.com/nodejs/Release/main/schedule.json)

[nodejs.dev](https://nodejs.dev/en/download/releases/) shows 0.8, 0.10, 0.12
But [nodejs.org](https://nodejs.org/en/download/releases) shows only 0.12

I am thinking of changing it to show only 0.12 so the behavior is consistent: show only the latest version for each major. What do you think?

Reply:
Only showing 0.12 is fine